src/util/BaseLoginRouter.js

```diff
@@ -199,9 +205,10 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
 
         if (!oldController) {
           this.el.append(this.controller.el);
+          this.controller.triggerEvent();
```

Comment:
This only gets called when the first page is rendered. I'm guessing you won't get these events when you navigate to the next page (i.e. if you had mfa enabled, you'd see an event on primary auth, but not on the mfa challenge page).